delta/controller.go

```diff
@@ -62,6 +62,7 @@ func NewController(
 		log:             log.WithField("component", "delta"),
 		client:          client,
 		delta:           newDelta(log, logLevel, stateProvider),
+		initialDelay:    60 * time.Second,
```

Comment:
60 seconds is quite long time. Just waiting for informers to sync won't work here?